.goreleaser.yml

```diff
@@ -51,6 +51,11 @@ dockers:
   image_templates:
   - 'kanisterio/postgres-kanister-tools:{{ .Tag }}'
   dockerfile: 'docker/postgres-kanister-tools/Dockerfile'
+- binaries:
+  - kando
+  image_templates:
+  - 'kanisterio/postgres-kanister-tools-9.6:{{ .Tag }}'
```

Comment:
Should we name this `kanisterio/postgres:{{ .Tag }}` or `kanisterio/postgres-tools:{{ .Tag }}` instead?
We already have an issue to have constant image name conventions https://github.com/kanisterio/kanister/issues/448

Reply:
Hi @PrasadG193 
We are not changing the image version right the version will be `:{{ .Tag }}` I am including `9.6` in the image name.
Btw this is what I and @tdmanv  discussed.

Reply:
I think `kanister` in the image name is redundant. How about `kanisterio/postgres-tools-9.6:{{ .Tag }}`?